src/workers/parser/mapExpression.js

```diff
@@ -30,22 +32,23 @@ export default function mapExpression(
     originalExpression: false
   };
 
+  const getAst = memoize(parseConsoleScript).bind(null, expression);
```

Comment:
do you need? `memoize(parseConsoleScript).bind(null, expression);` 

I feel like `getAst(expression)` would work just as well

Reply:
right, this can be moved to another PR

Reply:
I removed the `getAst` function and just put the result in an `ast` variable. We'll handle memoization in a new PR